fe/fe-core/src/main/java/com/starrocks/analysis/PauseRoutineLoadStmt.java

```diff
@@ -1,24 +1,11 @@
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
-//
-//   http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
If this file is inherited from origin codebase, please do not remove the apache license.

Reply:
[use the original license](https://github.com/StarRocks/starrocks/pull/8909/commits/a794892433e6b23f8514f0d917a8e0f5762ab3a4)
[a794892](https://github.com/StarRocks/starrocks/pull/8909/commits/a794892433e6b23f8514f0d917a8e0f5762ab3a4)